tracer/src/Datadog.Trace/Debugger/Snapshots/Redaction.cs

```diff
@@ -0,0 +1,292 @@
+// <copyright file="Redaction.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security;
+using System.Text;
+using Datadog.Trace.Debugger.Helpers;
+
+namespace Datadog.Trace.Debugger.Snapshots
+{
+    internal static class Redaction
+    {
+        private static readonly Type[] AllowedCollectionTypes =
+        {
+            typeof(List<>),
+            typeof(ArrayList),
+            typeof(LinkedList<>),
+            typeof(SortedList),
+            typeof(SortedList<,>),
+            typeof(Stack),
+            typeof(Stack<>),
+            typeof(ConcurrentStack<>),
+            typeof(Queue),
+            typeof(Queue<>),
+            typeof(ConcurrentQueue<>),
+            typeof(HashSet<>),
+            typeof(SortedSet<>),
+            typeof(ConcurrentBag<>),
+            typeof(BlockingCollection<>),
+            typeof(ConditionalWeakTable<,>),
+        };
+
+        private static readonly Type[] AllowedDictionaryTypes =
```

Comment:
This has nothing to do with redaction, so seems inappropriate that it's in this class?